src/includes/blocks/technology.pug

```diff
@@ -3,7 +3,7 @@
     .container
         h1 {{ page.title }}  (Technology)
         p {{ page.description }}
-        p {{ content }}
+        {{ content }}
```

Comment:
weird that jade doesn't pick it up

Webstorm did

![image](https://cloud.githubusercontent.com/assets/656739/25725764/4d79d6ac-3119-11e7-9ebd-3850bd050ca8.png)